src/components/ExportCapture.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Checkbox,
+  FormControl,
+  FormGroup,
+  FormControlLabel,
+} from '@material-ui/core';
+import { CapturesContext } from '../context/CapturesContext';
+import { CSVLink } from "react-csv";
+import { formatCell } from './CaptureTable';
+import { tokenizationStates } from 'common/variables';
+import { getDateTimeStringLocale } from 'common/locale';
+
+const useStyle = makeStyles((theme) => ({
+  container: {
+    position: 'relative',
+    padding: theme.spacing(0, 4),
+  },
+  textInput: {
+    margin: theme.spacing(2, 1),
+    flexGrow: 1,
+  },
+  formControl: {
+    margin: theme.spacing(3),
+  },
+  csvLink: {
+    textDecoration: "none",
+    color: theme.palette.stats.green,
+  }
+}));
+
+const ExportCapture = (props) => {
+  const { isOpen, handleClose, columns, filter, speciesState } = props;
+  const classes = useStyle();
+  let nameColumns = {};
+  columns.forEach(({ attr }) => {
+    nameColumns[attr] = true;
+  });
+  const [checkedColumns, setCheckColumns] = useState(nameColumns);
+  const capturesContext = useContext(CapturesContext);
+  const [downloadData, setDownloadData] = useState([]);
+
+  function handleChange(attr) {
+    setCheckColumns({ ...checkedColumns, [attr]: false });
+  }
+
+  function processDownloadData(captures, selectedColumns) {
+    return captures.map(capture => {
+      let formatCapture = {};
+      Object.keys(selectedColumns).forEach(attr => {
+        if (attr === 'id' || attr === 'planterId') {
+          formatCapture[attr] = capture[attr];
+        } else if (attr === 'tokenId') {
+          formatCapture[attr] = formatCell(capture, speciesState, attr, (val) =>
+          val ? tokenizationStates.TOKENIZED : tokenizationStates.NOT_TOKENIZED);
+        } else if (attr === 'timeCreated') {
+          formatCapture[attr] = formatCell(capture, speciesState, attr, (val) => getDateTimeStringLocale(val));
+        } else {
+          formatCapture[attr] = formatCell(capture, speciesState, attr, null);
+        }
+      })
+      return formatCapture;
+    })
+  }
+
+  return (
+    <Dialog
+      open={isOpen}
+      onClose={handleClose}
+      aria-labelledby="form-dialog-title"
+    >
+      <DialogTitle id="form-dialog-title">Export Captures</DialogTitle>
+      <DialogContent>
```

Comment:
Add a label with of the number of captures to be downloaded. This would be a good place to warn the user that only the first 30,000 will be downloaded.